nuklear.h

```diff
@@ -28262,13 +28262,18 @@ nk_combo_begin_text(struct nk_context *ctx, const char *selected, int len,
         struct nk_rect label;
         struct nk_rect button;
         struct nk_rect content;
+        int draw_button_symbol;
```

Comment:
@zgcrowno thanks for the effort and tracing it down.

Well, it's probably nitpicking, but why to introduce a new variable which actually doesn't vary?

It seems more straightforward to read `if( sym != NK_SYMBOL_NONE ){ ... }` than `if( draw_button_symbol ){ ... }` in the given contexts (with a variable I'd need to look up how it was defined and because it's **vari**able also how it got changed in the code flow, which contrasts with the simple (non-)equality check).

If you had a reason, just tell me and we'll happily merge it as otherwise the PR looks good to me :wink:.

Reply:
No problem, @dumblob. I've generally always used variables for values that are referenced several times, so as to both lessen the chance of mistyping the value, and to reinforce the idea that those sections of code which reference that value are somewhat related to one another. Having said that, this is a particularly straightforward case, so I'd be happy to change it if you'd like.